### ISSUE TITLE
sklearn's Imputer has been renamed & moved

### ISSUE THREAD

**Issue by rhiever** [EpistasisLab/tpot#889]
As of sklearn v0.21.2 (possibly earlier), the `Imputer` has moved to the `impute` module and is now called the `SimpleImputer`: https://scikit-learn.org/stable/modules/generated/sklearn.impute.SimpleImputer.html

TPOT likely needs a quick update to update this reference when we perform imputation at the beginning of every TPOT run.

Also worth noting that the `impute` module offers [several different imputation methods](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.impute) now, so perhaps it's worthwhile to move the imputation step into the configuration dictionary.

We can probably address #836 while making this change.

**Comment by weixuanfu** [comment]
I agree with this update. Also, for addressing this and also supporting future version of sklearn (> 0.22), TPOT should not support Python 2 any more.

**Comment by rhiever** [comment]
Agreed. It is time for future updates of TPOT to drop support for Python 2.
